modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3065,6 +3065,67 @@ definition {
 		}
 	}
 
+	@description = "LPS-173537 - Verify on the Account object if it's possible to trigger an action with ADD when ADD an account"
```

Comment:
Update the description to: `LPS-173537 - Verify creating an Account entry triggers an action to add a second Account entry`